server/asset/btc/btc.go

```diff
@@ -127,9 +130,19 @@ type Backend struct {
 	chainParams *chaincfg.Params
 	// A logger will be provided by the dex for this backend. All logging should
 	// use the provided logger.
-	log         dex.Logger
-	decodeAddr  dexbtc.AddressDecoder
-	estimateFee func(*RPCClient) (uint64, error)
```

Comment:
:heart: 
I'm looking to make a similar change in the future to client where we have the awkward issue of a `client/asset.Wallet` having to be a `RawRequester`, something that stands out given it's RPC-oriented.